content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -10,6 +10,8 @@ weight: 50
 
 
 <!-- overview -->
+Taints are a Kubernetes node property that enable nodes to repel certain pods. Tolerations are a Kubernetes pod property that allow a pod to be scheduled on a node with a matching taint.
+
```

Comment:
maybe we can also mention that _Taints and Toleratioins_ doesn't guarantee restriction of other Pod to a node in which taint has been applied.